### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by benparj98**
New content type sample

### CODE REVIEW

File: messaging/content-templates/create-carousel-template/create-carousel-template.curl [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    "friendly_name": "twilio_deal",
+    "language": "en",
+    "variables": {
+        "1": "Twilio"
+    },
+    "types": {
+        "twilio/carousel": {
+                    "body": "New {{1}} merch just dropped! 👀",
+  "cards": [
+    {
```
Comment (CristhianMotoche) [inline messaging/content-templates/create-carousel-template/create-carousel-template.curl:14]: Maybe "cards" need to be at the same level of "body"?
```suggestion
        "twilio/carousel": {
                 "body": "New {{1}} merch just dropped! 👀",
                 "cards": [
                     {
```

**Review by benparj98 — COMMENTED** [review]
lgtm

File: messaging/content-templates/create-carousel-template/output/create-carousel-template.json [hunk 2]
```diff
@@ -0,0 +1 @@
...s.", "media": "https://sienna-grasshopper-3262.twil.io/assets/hoodie.jpeg", "title": "Twilio Hoodie"}, {"actions": [{"id": "want_tote", "index": 0, "title": "I want it!", "type": "QUICK_REPLY"}, {"title": "Take the tote!", "type": "URL", "url": "https://sienna-grasshopper-3262.twil.io/assets/tote.jpeg"}], "body": "Carry a little more.", "media": "https://sienna-grasshopper-3262.twil.io/assets/tote.jpeg", "title": "Twilio Tote"}, {"actions": [{"id": "want_hat", "index": 0, "title": "I want it!", "type": "QUICK_REPLY"}, {"title": "Hand me the hat!", "type": "URL", "url": "https://sienna-grasshopper-3262.twil.io/assets/hat.jpeg"}], "body": "Stay in the shade.", "media": "https://sienna-grasshopper-3262.twil.io/assets/hat.jpeg", "title": "Twilio Bucket Hat"}, {"actions": [{"id": "want_mug", "index": 0, "title": "I want it!", "type": "QUICK_REPLY"}, {"title": "Make me a mug!", "type": "URL", "url": "https://sienna-grasshopper-3262.twil.io/assets/mug.jpeg"}], "body": "Sip a little.", "media": "https://sienna-grasshopper-3262.twil.io/assets/mug.jpeg", "title": "Twilio Mug"}]}}, "url": "https://content.twilio.com/v1/Content/HX2dcc3a9a5c8f4b428e3ada5a7e9dbee1", "variables": {"1": "Twilio"}}
```
Comment (bermudezmt) [inline messaging/content-templates/create-carousel-template/output/create-carousel-template.json:1]: Do you want to prettify this, @benparj98 ? https://jsonformatter.org/json-pretty-print

Actually @CristhianMotoche , does this automatically get fixed when ingested by the platform?

Comment (benparj98) [reply messaging/content-templates/create-carousel-template/output/create-carousel-template.json]: I just went ahead and made it pretty!

**Review by bermudezmt — APPROVED** [review]

**Pull request merged**